sections/recommendations.tex

```diff
@@ -33,11 +33,44 @@ \subsection*{Magic Arrows}
 Except for the Magic Arrow, each Spell appears exactly twice in the Spell Deck.
 After players prepare their starting M\&M Decks, it is recommended to remove Magic Arrows from the Spell Deck until no more than two remain.
 
+\subsection*{Guaranteed Settlement}
+
+When distributing Far Tiles (II--III) between players, each player's pool of Map Tiles must include one Settlement.
+
+\includegraphics[width=\linewidth]{\art/rogue.jpg}
+
+\includegraphics[width=\linewidth]{\art/medusa.jpg}
+
+\subsection*{War Machines as Artifacts}
```

Comment:
This hasn't been tested at all, by the way (hence the 'optional' flag in the TTS House Rules channel).

Reply:
We could mark them somehow as optional/not playetested, or remove them entirely once the playtest prove them unworthy. Which approach do you think is better?

Reply:
I'd add only tested and proven rules that we know work well